### PR TITLE
Test using random input

### PR DESCRIPTION
**Pull request by shellhub**
### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: sorts/bubble_sort.py [hunk 1]
```diff
@@ -34,10 +40,13 @@ def bubble_sort(collection):
...
 if __name__ == "__main__":
+    import doctest
     import time
 
+    doctest.testmod()
+
     user_input = input("Enter numbers separated by a comma:").strip()
     unsorted = [int(item) for item in user_input.split(",")]
-    start = time.process_time()
+    start = time.time()
```
Comment (cclauss) [inline sorts/bubble_sort.py:50]: Why make this change?

Comment (shellhub) [reply sorts/bubble_sort.py:50]: I like `time()` function. Do you think which is better.

Comment (cclauss) [reply sorts/bubble_sort.py:50]: I think that the original author used [`time.process_time()`](https://docs.python.org/3/library/time.html#time.process_time) for a reason.  And "I like a different function" is not enough of a reason to change the author's intention.

Comment (shellhub) [inline sorts/bubble_sort.py:50]:
```suggestion
    start = time.process_time()
```

File: sorts/bubble_sort.py [hunk 2]
```diff
@@ -34,10 +40,13 @@ def bubble_sort(collection):
...
     print(*bubble_sort(unsorted), sep=",")
-    print(f"Processing time: {time.process_time() - start}")
+    print(f"Processing time: {time.time() - start}")
```
Comment (shellhub) [inline sorts/bubble_sort.py:52]:
```suggestion
    print(f"Processing time: {process_time() - start}")
```

**Review by shellhub — COMMENTED** [review]
undo `time()`  changes

File: sorts/bubble_sort.py (hunk 1 above)
Comment (shellhub) [reply sorts/bubble_sort.py]: alright. I undo it. Please review again.

**Comment by shellhub** [comment]
@cclauss @poyea I this the PR can be merged to master. :)

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Pull request merged**